tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -128,9 +128,9 @@ bgp/test_bgp_slb.py:
 
 bgp/test_bgp_slb.py::test_bgp_slb_neighbor_persistence_across_advanced_reboot:
   skip:
-    reason: "Skip it on dual tor since it got stuck during warm reboot due to known issue on master and internal image"
+    reason: "Skip it on dual tor since it got stuck during warm reboot due to known issue on master and internal image; skip warm-reboot on SN4700 since it is not currently meant to be qualified."
     conditions:
-      - "topo_name in ['dualtor', 'dualtor-56', 'dualtor-120']"
+      - "(topo_name in ['dualtor', 'dualtor-56', 'dualtor-120']) or ('SN4700' in hwsku and 't0' in topo_type)"
```

Comment:
`and 't0' in topo_type` is not really needed. t1, t2 and other topos do not need warm-reboot anyway.

Just adding a skip for sn4700, regardless of topo_type is sufficient.